Remotion/Data/DomainObjects/Persistence/Rdbms/RdbmsProvider.cs

```diff
@@ -126,7 +144,7 @@ public virtual void Connect ()
       {
         _connection = new TracingDbConnection(CreateConnection(), PersistenceExtension);
         if (string.IsNullOrEmpty(_connection.ConnectionString))
-          _connection.ConnectionString = StorageProviderDefinition.ConnectionString;
+          _connection.ConnectionString = ConnectionString;
```

Comment:
somehow this slipped through in the review on develop. we should be using the connectionstring field, not the property. I don't think it would create a merge conflict if we fixed it here. then I might remmber patching it on develop, too. or you add a fixup commit so we remember to do it on develop during the rebase :)

Reply:
In the original RM-6996 branch, I created a commit with message "fixup f24f1d2 RM-9129 Add ReadOnlyConnectionString to IStorageObjectFactory" that fixes this.